observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -117,7 +117,15 @@ spec:
 [source,shell]
 ----
 $ oc adm policy add-cluster-role-to-user collect-application-logs -z collector -n openshift-logging
+----
++
+[source,shell]
+----
 $ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector -n openshift-logging
+----
++
+[source,shell]
+----
```

Comment:
A global comment because I see that there are other instances of this rule in this file beyond the PR changes:

According to the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output), we have to use `[source,terminal]` for CLI command blocks:
```suggestion
+
[source,terminal]
----
$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector -n openshift-logging
----
+
[source,terminal]
----
```